Edison POI tweaks

PeccNeck opened the pull request:
## About the PR
- Adds vacuum markers everywhere. Especially in that weird gas chamber at the sale point.
- Fixes distro. It's now set to the correct 78/22 instead of default 50/50.
- Replaces the welding fuel locker in the SMES room with one that's actually filled (for the JRPACMAN).
- Unhides all-access base variant of the lockable button from spawn menu.
- Replaces the lockable buttons for the burn chambers by the TEG with the aforementioned all-access buttons.
- Adds additional (ordinary) buttons inside burn chambers as emergency escapes in case someone locks you in.
- A couple airlocks now cycle correctly.

## Why / Balance
- Roundstart air mix would taint your sale if you forgot to take care of it first.
- Breathable.
- Both the locker and generator are somewhat useless without fuel (the fuel-less version isn't even mapable).
- Solely for the below.
- Makes public area burn chambers actually publicly accessible.
- Self-explanatory.
- Edison has airlocks without directional fans. This seems to have been a deliberate design decision, as most airlocks were already set up to bolt; a couple didn't and would space easily, this fixes that.

## Technical details
Map edits, YML change.

## How to test
- Sell gas. Notice no air mix impurity.
- Space room. Refill room. Air mix is actually air mix.
- Commit breaking and entering. Refuel the JRPACMAN from the locker.
- Search "lockable button" in Spawn Entities. See "lockable button [directional]" at bottom.
- Open "lockable" cover to one of the west burn chambers, then open said chamber.
- Lock yourself into the burn chamber you just opened. Then free yourself.
- Enter airlock. Wait for it to cycle. Exit airlock.

## Media
It's like, 3 buttons and a bajillion vacuum markers, media seems excessive.

## Requirements
- [x] I have read [CONTRIBUTING.md](https://github.com/new-frontiers-14/frontier-station-14/blob/master/CONTRIBUTING.md) and and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
- [x] I confirm that AI tools were not used in generating the material in this PR.

**Changelog**
:cl:
- tweak: Fixed a bunch of little things at Edison Power Plant

Cheackraze commented:
the no-fan airlock system in the atmos area is indeed intentional, designed for an atmosian to vent their working space in the event of a spicy pipe leak

the-hivequeen commented:
I understand the logic behind it, but in practice the no fans make the place a deathtrap. The railing on the outside only catches people getting shot through about half the time, so it consistently just railguns people into space to their deaths. I've seen 0 people die there from spicy air (and there are axes and maintenance jacks available on hand) but at least fifteen people die to the fans not being there.

Qiler commented:
> the no-fan airlock system in the atmos area is indeed intentional, designed for an atmosian to vent their working space in the event of a spicy pipe leak

Perhaps a shutter on a wall designed to vent the room would be better, there is a good place for it right next to axe case.

PeccNeck commented:
> > the no-fan airlock system in the atmos area is indeed intentional, designed for an atmosian to vent their working space in the event of a spicy pipe leak
> 
> Perhaps a shutter on a wall designed to vent the room would be better, there is a good place for it right next to axe case.

Modified to include a pair of shutters in front of the waste chamber. Opted to put it there for readability; next to the axe leaves no real room for another button and could easily confuse one button for the other.
![image](https://github.com/user-attachments/assets/b53bb7ca-eda4-4d51-a416-f327c6c4c872)

Also removed like 90% of the vacuum markers I added the first go around; many were redundant and didn't accomplish anything that wasn't already done (except hurt readability).

> I understand the logic behind it, but in practice the no fans make the place a deathtrap.

I'm still gonna refrain on adding directional fans as they're not seen on any of the other airlocks (except the docks). The doors bolting should _help_, even if they're not perfect.

PeccNeck commented:
> Hi!
> 
> I wanted to check in to see where you felt you were with this update. Are you feeling confident that it is currently in a state where you're feeling confident or are you still making adjustments?

Unless there's any other critiques I think this is in a position where I'm fine with it. The only remaining concern from the original set is the inherent space wind of a proper, non-fanned airlock, but I think that's stepping over a line of "QOL tweaks" to "design philosophy" and something that should be addressed separately (not here).
